ci.nix

```diff
@@ -64,6 +64,7 @@ mkShell {
   PROTOC_INCLUDE = io-engine.PROTOC_INCLUDE;
   SPDK_PATH = if nospdk then null else "${libspdk-dev}";
   ETCD_BIN = "${etcd}/bin/etcd";
+  FIO_SPDK = "${libspdk-dev}/fio/spdk_nvme";
```

Comment:
I wonder if we need to check `if no spdk then null` like SPDK_PATH, in case we're using custom spdk?

Reply:
ah, yes. Will fix.